network.go

```diff
@@ -1089,8 +1083,19 @@ func (n *network) delete(force bool, rmLBEndpoint bool) error {
 	c.cleanupServiceDiscovery(n.ID())
 
 	// Cleanup the load balancer. On Windows this call is required
-	// to remove remote loadbalancers in VFP.
-	c.cleanupServiceBindings(n.ID())
+	// to remove remote loadbalancers in VFP, and must be performed before
+	// dataplane network deletion.
+	if runtime.GOOS == "windows" {
+		c.cleanupServiceBindings(n.ID())
+	}
```

Comment:
This is not needed on Linux? Is it needed in other situations? e.g., I see it's called in `clusterAgentInit()`;  that's still needed for Linux?  (Was thinking if we'd need different implementations for Linux and Windows)

https://github.com/moby/libnetwork/blob/b3507428be5b458cb0e2b4086b13531fb0706e46/controller.go#L349-L358